perf/specjbb/run/run_multi.sh

```diff
@@ -0,0 +1,80 @@
+#!/bin/bash
```

Comment:
Just noting for now incase we wanted to add a powershell version of this file (thank you ChatGPT)

```powershell
Write-Output "Backend opts: $env:JAVA_OPTS_BE"
Write-Output "Number of runs: $env:NUM_OF_RUNS"
Write-Output "Results dir: $env:RESULTS_DIR"

for ($n=1; $n -le $env:NUM_OF_RUNS; $n++) {

  # Create result directory                
  $temp = $env:RESULTS_DIR.Trim('"')
  $timestamp = Get-Date -Format 'yy-MM-dd_HHmmss'
  $result = "$temp/$timestamp"
  New-Item -ItemType Directory -Path $result -Force | Out-Null

  # Copy current config to the result directory
  Copy-Item $env:SPECJBB_CONFIG $result -Recurse

  Set-Location $result

  Write-Output "Run $n: $timestamp"
  Write-Output "Launching SPECjbb2015 in MultiJVM mode..."
  Write-Output

  $JAVA = $env:JAVA.Trim('"')
  Write-Output "Start Controller JVM"
  Start-Process -FilePath $JAVA -ArgumentList "$env:JAVA_OPTS_C $env:SPEC_OPTS_C -jar $env:SPECJBB_JAR -m MULTICONTROLLER $env:MODE_ARGS_C" -RedirectStandardError controller.log -RedirectStandardOutput controller.out -NoNewWindow -PassThru | Out-Null
  $CTRL_PID = $LASTEXITCODE
  Write-Output "Controller PID = $CTRL_PID"

  Start-Sleep -Seconds 3

  for ($gnum=1; $gnum -lt $env:GROUP_COUNT + 1; $gnum++) {

    $GROUPID = "Group$gnum"
    Write-Output "`nStarting JVMs from $GROUPID:"

    for ($jnum=1; $jnum -lt $env:TI_JVM_COUNT + 1; $jnum++) {

        $JVMID = "txiJVM$jnum"
        $TI_NAME = "$GROUPID.TxInjector.$JVMID"

        Write-Output "    Start $TI_NAME"
        Start-Process -FilePath $JAVA -ArgumentList "$env:JAVA_OPTS_TI $env:SPEC_OPTS_TI -jar $env:SPECJBB_JAR -m TXINJECTOR -G=$GROUPID -J=$JVMID $env:MODE_ARGS_TI" -RedirectStandardError $TI_NAME.log -RedirectStandardOutput $TI_NAME.log -NoNewWindow -PassThru | Out-Null
        Write-Output "`t$TI_NAME PID = $LASTEXITCODE"
        Start-Sleep -Seconds 1
    }

    $JVMID = "beJVM"
    $BE_NAME = "$GROUPID.Backend.$JVMID"

    # Add GC logging to the backend's JVM options 
    $JAVA_OPTS_BE_WITH_GC_LOG = "$env:JAVA_OPTS_BE -Xlog:gc*:file=$BE_NAME
```